### PR TITLE
Added Jenkins override for EMAIL_ID_FOR_RESULTS_SHEET var

### PR DESCRIPTION
**Pull request by nathan-weinberg**
As discussed with @paigerube14

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -242,7 +248,9 @@ pipeline {
                 currentBuild.description = "Copying Artifact from Flexy-install build <a href=\"${buildinfo.buildUrl}\">Flexy-install#${params.BUILD_NUMBER}</a>"
                 buildinfo.params.each { env.setProperty(it.key, it.value) }
             }
-
+            if (params.EMAIL_ID_OVERRIDE != '') {
+                env.EMAIL_ID_FOR_RESULTS_SHEET = params.EMAIL_ID_FOR_RESULTS_SHEET
```
Comment (paigerube14) [inline Jenkinsfile:252]: Think you used the wrong variable, should be below?

env.EMAIL_ID_FOR_RESULTS_SHEET = params.EMAIL_ID_OVERRIDE

Comment (nathan-weinberg) [reply Jenkinsfile:252]: Whoops you're right, good catch!

**Comment by nathan-weinberg** [comment]
Verified with below jobs:
- With override: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/kube-burner/5/
- Without override: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/kube-burner/6/

**Pull request merged**